module/Finna/src/Finna/Controller/RecordController.php

```diff
@@ -229,7 +229,10 @@ protected function loadRecord(ParamBag $params = null, bool $force = false)
         }
         $recordFactory = $this->serviceLocator
             ->get(\VuFind\RecordDriver\PluginManager::class);
-        return $this->driver = $recordFactory->getSolrRecord($metadata);
+        $this->driver = $recordFactory->getSolrRecord($metadata);
+        $translator = $this->serviceLocator->get(\Laminas\Mvc\I18n\Translator::class);
```

Comment:
Parempi kaivaa LocaleSettings:iltä:
`$this->serviceLocator->get(\VuFind\I18n\Locale\LocaleSettings::class)->getUserLocale()`

Reply:
jeppis!